readme.md

```diff
@@ -25,7 +38,23 @@ Your Answer:
 Describe at a high level how we use jQuery to submit a form via AJAX.
 
 Your Answer:
-```text
+```  
+$(".test_ajax_post").on("click", function(){
```

Comment:
So this code is along the right track, but what I'd like to see here is a high-level description of what it's doing, and why it's doing it.